src/Makefile.am

```diff
@@ -12,9 +12,6 @@ ynclude_DATA =			\
 	dialogs.py		\
 	complex.py
 
-aduc.desktop: org.opensuse.yast.ADUC.desktop.in
```

Comment:
Previous changes broke the build, because line 16 now creates org.opensuse.yast.ADUC.desktop instead of aduc.desktop (required by line 15).
This is old, unnecessary cruft anyway. Just remove it.